### ISSUE TITLE
Nemo in Mint Cinnamon not showing right number of selected files in folder with lots of pictures

### ISSUE THREAD

**Issue by claudemontreal** [linuxmint/nemo#2469]
```
 * Nemo version (nemo --version) 4.6.4
 * Is issue with desktop or windowed nemo? not sure what this means
 * Distribution - (Mint 17.2, Arch, Fedora 25, etc...) Mint 20
 * Graphics hardware *and* driver used Intel HD Graphics 5500 driver i915
 * 32 or 64 bit 64 bit
 ```

**Issue**
In a folder with a large number of pictures, when selecting several files, Nemo will show the accurate number files for a fraction of a second, then show an inaccurate number.

In this screenshot, Nemo says that there are 17 files selected, while in fact there are 42 files selected.

![Capture d’écran du 2020-08-02 11-38-36](https://user-images.githubusercontent.com/69161793/89205725-27e6ea00-d586-11ea-8bec-7b21ad4249f8.png)

**Steps to reproduce**
Go in a folder with a large number of pictures (a few hundreds) and select a part of them (50 or more)

**Expected behaviour**
It should display the exact number of files selected.

**Other information**

**Comment by nrikonomov** [comment]
I already explained the problem in the LM20 blog post, copy-paste here:

Annoying bug in Nemo – might report to bug forums later – but I want to describe it not to forget it — so there is a folder containing 33 folders and 66 files. I press Ctrl-A and all of them are selected and the status bar reports 33 folders/66 other items. Then I use the mouse scroll while everything is selected and the number of items changes – now it says 15 folders/63 items – when I use right-click – compress – it zips only these files/folders, even though everything is marked. To fix this – I have to Ctrl-click on a file, which refreshes the file count, and Ctrl-click on a folder to refresh its counter. If I compress one-level higher, everything is fine. Reproducible – always – you can test in /usr/bin/ – Ctrl-A and then mouse-scroll.

Another example - /usr/bin/ - select 24 files with mouse and Shift, scroll with the mouse scroll, and now it says 14 items - if you zip (or copy) that - it will zip only 14 items, even though 24 are selected.

Affected Nemo version: 4.6.4+ulyana
Not affected Nemo version: 4.4.2 (LM19.3)
So the bug was introduced between these versions somewhere.

This is problematic, since I can't rely on Nemo copying/zipping my data properly.

**Comment by mtwebster** [comment]
This is a regression caused by delayed loading of thumbs that was implemented for 4.6.  I'm trying to come up with a solution that doesn't involve a performance hit.  The thumbnails are loaded or generated as you scroll.

The update to files as their thumbnail finishes loading is causing the selection list to be corrupted.  So, this won't happen if you scroll thru the file list completely to allow all images to load before making selections.  Yes this is annoying I know - I'm trying to fix it asap.

**Comment by Jeremy7701** [comment]
Also present in nemo for LMDE4.
